actnow/urls.py

```diff
@@ -16,11 +16,14 @@
 from django.contrib import admin
 from django.urls import include, path
 
+from .metrics import metrics
+
 urlpatterns = [
     path("admin/", admin.site.urls),
-    path("accounts/", include("actnow.accounts.urls")),
-    path("petitions/", include("actnow.petitions.urls")),
-    path("profiles/", include("actnow.profiles.urls")),
+    path("v1/accounts/", include("actnow.accounts.urls")),
+    path("v1/metrics/", metrics, name="metrics"),
```

Comment:
This one should also just be empty ie just `/v1/`

Reply:
Not sure. Was thinking having it as `v1/metrics/` makes it a little bit more descriptive no?

Reply:
Go on, why do you think that?

Reply:
Nothing more. Just that the endpoint describes what data it's providing.

Reply:
Yes and being on `/v1` it describes/gives snapshot of the whole `/v1`

Reply:
sure. that makes sense too.